dix/req.c

```diff
@@ -2153,7 +2224,26 @@ static int dix_cas_rops_alloc(struct m0_dix_req *req)
 	if (cas_rop_tlist_is_empty(&rop->dg_cas_reqs))
 		return M0_ERR(-EIO);
 
+	if (dtx != NULL) {
+		M0_ASSERT(!req->dr_is_meta);
+		M0_ASSERT(M0_IN(req->dr_type, (DIX_PUT, DIX_DEL)));
+		rc = m0_dtx0_open(dtx, cas_rop_tlist_length(&rop->dg_cas_reqs));
+		if (rc != 0)
+			goto end;
+
+		i = 0; /* reset it to enumerate dtx parcticipants */
```

Comment:
1. I prefer to use another variable instead of re-use 'i'.
2. If you want to re-use this variable, we can move this reset out of this condition block to the position just before the m0_tl_for(). That will unconditionally reset it before re-use it.

Reply:
s/parcticipants/participants

Reply:
s/parcticipants/participants

Reply:
s/parcticipants/participants

Reply:
Fixed.